applications/butler/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
```

Comment:
Naive question but why is this v2? We haven't done v1 yet.

Reply:
That's the Helm chart minimum version number, it's not related to our app's version in any way.